kidash/kidash.py

```diff
@@ -1118,8 +1127,8 @@ def get_dashboard_name(panel_file):
     if kibana and 'dashboard' in kibana:
         dash_name = kibana['dashboard']['id']
     elif kibana:
-        logger.error("Wrong panel format (can't find 'dashboard' or 'index_patterns' fields): %s",
-                     panel_file)
+        logger.error("Wrong panel format (can't find 'dashboard' or "
+                     "'index_patterns' fields): %s", panel_file)
```

Comment:
instead of such a format, we can adapt a different multiline too like
```python
        logger.error("""Wrong panel format (can't find 'dashboard' or 
                     index_patterns' fields): %s""", panel_file)
```

WDYT?

Reply:
Multiline format will add a new line and tabs at the beginning of the second line, so it will be hard to parse in case we need it.